src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Dashboard.vue

```diff
@@ -51,16 +51,7 @@
                 <v-row v-if="applications && applications?.length > 0 && applicationStore.hasDraftApplication">
                   <v-col>
                     <v-btn variant="flat" rounded="lg" color="primary" @click="$router.push('/application')">Continue Your Application</v-btn>
-                    <ConfirmationDialog
-                      :config="{ cancelButtonText: 'Keep Application', acceptButtonText: 'Cancel Application', title: 'Cancel Application' }"
-                      @accept="cancelApplication"
-                    >
-                      <template #activator>Cancel Application</template>
-                      <template #confirmation-text>
-                        <p>By cancelling your application, it will be removed from the system. You cannot undo this.</p>
-                        <p><b>Are you sure you want to proceed?</b></p>
-                      </template>
-                    </ConfirmationDialog>
+                    <v-btn rounded="lg" variant="outlined" @click="showCancelDialog = true">Cancel Application</v-btn>
```

Comment:
not sure where to put this. But when you get the dialog and click outside of it the dialog will close. 

But then it won't open up again afterwards until we refresh the page.

Reply:
Did some debugging, seems to happen cause we don't set the showDialog prop to false when we click outside the modal. 

Could make the dialog "persistent" so clicks outside do not close it. 

Or hook into the @click:outside to emit the "cancel" event.